### ISSUE TITLE
Cant Add Views loaded from other view controllers or even views from view controller it self

### ISSUE THREAD

**Issue by AbdallahJaja** [aryaxt/ScrollPager#31]
Cant Add Views loaded from other view controllers or even views from view controller it self

**Comment by aryaxt** [comment]
can you explain how you can't do it? What's the problem you are having?

**Comment by QuocLoi** [comment]
Can it load ViewControllers instead of only UIViews?
I want to load multi view controllers when tapping on different tabs but it looks like impossible.

**Comment by aryaxt** [comment]
@QuocLoi  yes simply add the view of the viewController to ScrollPager, and make sure you keep a reference to the controller to ensure it doesn't get released

**Comment by deadpool9987** [comment]
@aryaxt  you can't add the view of the viewcontroller to scroll pager , even if i keep the refrence to the controller

**Comment by deadpool9987** [comment]
@aryaxt My Bad it is possible to add the views of the other viewcontroller and i have achieved the same.

**Comment by dilipiOSDeveloper** [comment]
how to show ViewController's on scrollpager
